01_network_sysapi/Client-Server_Mouse_check_app/modules/network/socket.h

```diff
@@ -0,0 +1,33 @@
+#ifndef SOCKET_H
+#define SOCKET_H
+
+#ifdef linux
```

Comment:
Even if you decided not to support another platforms it would be better to handler such cases by generating error.